receiver/vcenterreceiver/client.go

```diff
@@ -236,11 +237,16 @@ func (vc *vcenterClient) ResourcePoolInventoryListObjects(ctx context.Context) (
 // VAppInventoryListObjects returns the vApps (with populated InventoryLists) of the vSphere SDK
 func (vc *vcenterClient) VAppInventoryListObjects(ctx context.Context) ([]*object.VirtualApp, error) {
 	vApps, err := vc.finder.VirtualAppList(ctx, "*")
-	if err != nil {
-		return nil, fmt.Errorf("unable to retrieve vApps with InventoryLists: %w", err)
+	if err == nil {
+		return vApps, nil
+	}
+
+	var notFoundErr *find.NotFoundError
+	if errors.As(err, &notFoundErr) {
```

Comment:
Does `errors.Is(err, find.NotFoundError)` accomplish the same thing? If so it's a little more concise and doesn't require an allocation.

Reply:
@djaglowski I only used errors.As because the linter specifically told me to 😄

Reply:
Ah, ok. Sounds good then